<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="collaberas-mbp" failures="1" tests="10" name="Launch Test" time="279.452" errors="0" timestamp="2023-04-20T01:08:18 PST">
  <testcase classname="test.TestActivityDay11" name="verifyLoginTestMethod" time="15.501"/>
  <testcase classname="test.TestActivityDay11" name="verifyAccountMethod" time="11.268"/>
  <testcase classname="test.TestActivityDay11" name="verifyOrdersMethod" time="12.398"/>
  <testcase classname="test.TestActivityDay11" name="verifyBrowsingHistoryTestMethod" time="14.123"/>
  <testcase classname="test.TestActivityDay11" name="verifyCustomerServiceTestMethod" time="7.837"/>
  <testcase classname="test.TestActivityDay11" name="verifySearchTestMethod" time="19.306"/>
  <testcase classname="test.TestActivityDay11" name="verifyAddProductTestMethod" time="35.595"/>
  <testcase classname="test.TestActivityDay11" name="verifyDeleteProductTestMethod" time="50.567"/>
  <testcase classname="test.TestActivityDay11" name="verifyAddTwoProductTestMethod" time="76.187"/>
  <testcase classname="test.TestActivityDay11" name="verifyLogoutTestMethod" time="25.165">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at test.TestActivityDay11.verifyLogoutTestMethod(TestActivityDay11.java:430)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- verifyLogoutTestMethod -->
</testsuite> <!-- Launch Test -->
